cinema/urls.py

```diff
@@ -1,10 +1,28 @@
 from django.urls import path
 
-from cinema.views import movie_list, movie_detail
+from cinema.views import genre_list, genre_detail, ActorList, ActorDetail, CinemaHallDetail, CinemaHallList, MovieViewSet
+
+movie_list = MovieViewSet.as_view(actions={
+            "get": "list",
+            "post": "create",
+        })
+
+movie_detail = MovieViewSet.as_view(actions={
+            "get": "retrieve",
+            "put": "update",
+            "patch": "partial_update",
+            "delete": "destroy",
+        })
 
 urlpatterns = [
-    path("movies/", movie_list, name="movie-list"),
-    path("movies/<int:pk>/", movie_detail, name="movie-detail"),
+    path("api/cinema/genres/", genre_list, name="genre-list"),
+    path("api/cinema/genres/", genre_detail, name="genre-detail"),
+    path("api/cinema/actors/", ActorList.as_view(), name="actor-list"),
+    path("api/cinema/actors/<int:pk>/", ActorDetail.as_view(), name="actor-detail"),
+    path("api/cinema/cinamehalls/<int:pk>/", CinemaHallDetail.as_view(), name="cinemahall-detail"),
```

Comment:
There is a typo in the path for `CinemaHallDetail`. It should be `cinemahalls` instead of `cinamehalls`.